src/ucp/proto/proto_am.inl

```diff
@@ -346,9 +327,11 @@ ucs_status_t ucp_do_am_zcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
                 status = uct_ep_am_zcopy(uct_ep, am_id_middle, (void*)hdr_middle,
                                          hdr_size_middle, iov, iovcnt, 0,
                                          &req->send.state.uct_comp);
-            } else if (state.offset == req->send.length) {
+            } else if (req->send.state.dt.offset == req->send.length) {
```

Comment:
why change to `req->send.state`?

Reply:
returned back
doesn't matter which value is used here